### PR TITLE
GUI: Screens, widgets, huds and the things they call

### PR DESCRIPTION
**Pull request by skyrising**

### CODE REVIEW

**Review by hYdos — APPROVED** [review]

**Comment by BoogieMonster1O1** [comment]
iapprove

**Comment by BoogieMonster1O1** [comment]
i shall merge after checks complete because yes

**Comment by BluCobalt** [comment]
but is it called loadingScreen upstream

**Review by BluCobalt — APPROVED** [review]
resolved on discord

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 1]
```diff
@@ -72,7 +73,7 @@ CLASS net/minecraft/class_1321 net/minecraft/client/MinecraftClient
 	FIELD field_26892 textRenderer Lnet/minecraft/class_5575;
 	FIELD field_26893 enchantmentTextRenderer Lnet/minecraft/class_5575;
 	FIELD field_26894 currentScreen Lnet/minecraft/class_5641;
-	FIELD field_26895 splashScreen Lnet/minecraft/class_1324;
+	FIELD field_26895 loadingScreen Lnet/minecraft/class_1324;
```
[the remark this answers is not in the document]
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/MinecraftClient.mapping:76]: note: all discussion related to pull request reviews should happen on github as it serves as a record

[the remark this answers is not in the document]
Comment (BluCobalt) [reply mappings/net/minecraft/client/MinecraftClient.mapping:76]: image

**Pull request merged**